### DISCUSSION TITLE
Support for several untagged vlans

### DISCUSSION THREAD

**Discussion by kovalev94** [nautobot/nautobot#1045] [Ideas]
Hello. I think that interfaces 8021q modes should be a little bit changed, this is about only one untagged vlan limitation.
For example on my network i have Huawei campus switches (S2300 series) with Multicast Vlan Replication(Registration) setting. One part of this settings is set user port to hybrid mode with 2 untagged vlans - user vlan and multicast vlan, and setup pvid as user vlan. Another example is assymetric vlans  on L2 devices for sharing some resources(FTP,Printers, Web servers) with saving user isolation. 2 untagged vlans can exist on interface, and i think there are more cases.

**Comment by Niogui** [comment]
+1 Really need multiple untagged vlans and an Hybrid mode option for 802.1Q

**Comment by bryanculver** [comment]
Can you provide some link to documentation for "Hybrid mode option" for 802.1Q.

In our research most vendors use that to describe what we call "Tagged" or "Tagged (All)". Both allow optionally specifying an untagged VLAN but either explicitly stating the supported tagged VLANs (the former, "Tagged"), or implicitly allowing all known tagged VLANs (the latter, "Tagged (All)").

**Reply by Niogui** [reply]
Maybe here https://support.hpe.com/hpesc/public/docDisplay?docId=mmr_kc-0102233
or here https://arubase.club/wp-content/uploads/2019/05/CLI_Reference_Guide_for_ArubaOS-CX_ArubaOS-Switch_Comware_and_Cisco_IOS.pdf (page 6)
